README.md

```diff
@@ -34,11 +34,21 @@ Feel free to try out and let us know if you cannot wait.
 
 ## Install dockert3kit into your distribution
 
-Add `lauri/dockert3kit` as dev dependency in your composer, using the latest stable release is highly recommended.
+Add `lauri/dockert3kit` as dev dependency in your composer.
+
+***Note***
+
+**If you are using EXT:solr = 6.0.0 please use the 6.0.0 release**
+
+*Example*:
+
+    composer require --dev lauri/dockert3kit '~6.0.0'
+
+**If you are using EXT:solr = 6.1.0 please use the 6.1.0 release**
 
 *Example*:
 
-    composer require --dev lauri/dockert3kit '~3.0.0'
+    composer require --dev lauri/dockert3kit '~6.1.0'
```

Comment:
@yosokus 

https://github.com/laurisaarni/solr-for-typo3/ repository could follow versioning of Solr

This repository contains more than just Solr. This one has Nginx, Maria DB etc. So it's versioning should be independent of these and highlight changes on a more generic level.